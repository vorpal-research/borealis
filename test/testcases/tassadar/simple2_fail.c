
#include <string.h>
#include <stdlib.h>

int borealis_nondet();


int main() {
	int a[220] = {0};
	int x[22000];

	a[0] = 42;
	a[2] = 1;
	a[3] = 2;

	a[219] = 4;

	a[218] = 249;


	int* b = malloc(250*sizeof(int));

	b[42] = 218;

	b[a[b[42]] + 1] = a[219];

	return b[borealis_nondet()];
}