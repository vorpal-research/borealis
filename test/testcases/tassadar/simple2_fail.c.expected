[
"Memory write violation at 0x8001003e8"
]
